src/SolutionAcceleration.f90

```diff
@@ -0,0 +1,395 @@
+ENDTYPE ModifiedPicardType
+
+!> Name of module
+CHARACTER(LEN=*),PARAMETER :: modName='SolutionAccelerationModules'
+!
+!===============================================================================
+CONTAINS
+!
+!-------------------------------------------------------------------------------
+!> @brief Initializes the Solution Acceleration base class
+!> @param solver The Solution Acceleration solver to act on
+!> @param Params The options parameter list
+!> @param ce The computing environment to use for the calculation
+!>
+SUBROUTINE init_base_SolutionAccelerationType(solver,ce,Params)
+  CLASS(SolutionAccelerationType),INTENT(INOUT) :: solver
+  TYPE(FutilityComputingEnvironment),TARGET,INTENT(IN) :: ce
+  TYPE(ParamType),INTENT(IN) :: Params
+
+  CHARACTER(LEN=*),PARAMETER :: myName='init_base_SolutionAccelerationType'
+
+  REQUIRE(Params%has('SolutionAccelerationType->n'))
+  REQUIRE(.NOT.solver%isInit)
+
+  !Pull Data from Parameter List
+  CALL Params%get('SolutionAccelerationType->n',solver%n)
+  IF(Params%has('SolutionAccelerationType->start')) &
+      CALL Params%get('SolutionAccelerationType->start',solver%start)
+
+  IF(solver%n < 1) CALL ce%exceptHandler%raiseError('Incorrect input to '//modName// &
+      '::'//myName//' - Number of unkowns (n) must be greater than 0!')
+
+  IF(solver%start < 1) CALL ce%exceptHandler%raiseError('Incorrect input to '//modName// &
+      '::'//myName//' - Starting point must be greater than 1!')
+
+  solver%s=0
+  solver%ce => ce
+ENDSUBROUTINE init_base_SolutionAccelerationType
+!
+!-------------------------------------------------------------------------------
+!> @brief Clears the solution acceleration base class
+!> @param solver The solver to act on
+!>
+SUBROUTINE clear_base_SolutionAccelerationType(solver)
+    CLASS(SolutionAccelerationType),INTENT(INOUT) :: solver
+
+    INTEGER(SIK) :: i
+
+    IF(solver%isInit) THEN
+      IF(ALLOCATED(solver%x)) THEN
+        DO i=1,SIZE(solver%x)
+          CALL solver%x(i)%clear()
+        ENDDO
+        DEALLOCATE(solver%x)
+      ENDIF
+      solver%s=0
+      solver%n=-1
+      solver%start=1
+    ENDIF
+  ENDSUBROUTINE clear_base_SolutionAccelerationType
+!
+!-------------------------------------------------------------------------------
+!> @brief Initializes the relaxed picard type
+!> @param solver The relaxed picard solver to act on
+!> @param Params The options parameter list
+!> @param ce The computing environment to use for the calculation
+!>
+SUBROUTINE init_RelaxedPicardType(solver,ce,Params)
+  CLASS(RelaxedPicardType),INTENT(INOUT) :: solver
+  TYPE(FutilityComputingEnvironment),TARGET,INTENT(IN) :: ce
+  TYPE(ParamType),INTENT(IN) :: Params
+
+  CHARACTER(LEN=*),PARAMETER :: myName='init_RelaxedPicardType'
+
+  REQUIRE(.NOT.solver%isInit)
+
+  CALL solver%init_base(ce,Params)
+
+  IF(Params%has('RelaxedPicardType->alpha')) &
+      CALL Params%get('RelaxedPicardType->alpha',solver%alpha)
```

Comment:
I'm ok with this being optional... is `1.0` the right default though?

Reply:
I think so, it defaults to a standard Picard iteration.  0 is not an option.  You could maybe argue 0.5, but I don't have a huge preference.

Reply:
yeah, ok, I'm good with a non-relaxed Picard iteration being the default.  Just wanted to ask.